control-plane/CMakeLists.txt

```diff
@@ -51,8 +51,8 @@ set (P4RUNTIME_GEN_PYTHON "control-plane")
 add_custom_target (mkP4configdir
   ${CMAKE_COMMAND} -E make_directory ${CMAKE_BINARY_DIR}/p4/config)
 add_custom_command(OUTPUT ${P4RUNTIME_GEN_SRCS} ${P4RUNTIME_GEN_HDRS}
-  COMMAND ${PROTOBUF_PROTOC_EXECUTABLE}
-          -I ${P4RUNTIME_STD_DIR} -I${CMAKE_CURRENT_SOURCE_DIR}
+  COMMAND ${Protobuf_PROTOC_EXECUTABLE}
+          -I${P4RUNTIME_STD_DIR} -I${CMAKE_CURRENT_SOURCE_DIR} -I${Protobuf_INCLUDE_DIR}
```

Comment:
Why not `PROTOC_BINARY`?

Reply:
Using the generator expression is fine for commands in the build stage but not when writing to a file in the configure stage. There are some scripts (primarily in P4Testgen), which require access to the binary in the configure stage to write a file. I wish I had a good way to resolve this.

Reply:
I see. I've just noticed this is not consistent among all the usages, but I guess it actually makes sense to use the "canonical" way (provided by FindPackage) wherever possible.